pkg/client/openshift/buildconfig_test.go

```diff
@@ -16,7 +16,7 @@ package openshift
 
 import (
 	"context"
-	"github.com/kiegroup/kogito-cloud-operator/pkg/test"
+	test2 "github.com/kiegroup/kogito-cloud-operator/core/test"
```

Comment:
why naming this import?

Reply:
My IDE done this to me. I had fixed it now